cypress/integration/group2/categories.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { resetDB, setTokenUserViewPort, addOrganizationAdminUser } from '../../support/commands';
+
+describe('Dockstore Categories', () => {
+  const categoryName = 'foooo';
+  const toolPath = '/containers/quay.io/garyluu/dockstore-cgpmap/cgpmap-cramOut:3.0.0-rc8?tab=info';
+  const toolSnippet = 'cgpmap-cramOut';
+
+  resetDB();
+  setTokenUserViewPort();
+
+  function typeInInput(fieldName: string, text: string) {
+    cy.contains('span', fieldName).parentsUntil('.mat-form-field-wrapper').find('input').first().should('be.visible').clear().type(text);
+  }
+
+  describe('Should be able to create a category', () => {
+    it('be able to create a category', () => {
+      addOrganizationAdminUser('dockstore', 'user_A');
+      cy.visit('/organizations/dockstore');
+      cy.get('#createCollection').click();
+      cy.get('#createOrUpdateCollectionButton').should('be.visible').should('be.disabled');
+      typeInInput('Name', categoryName);
+      typeInInput('Display Name', categoryName);
+      typeInInput('Topic', 'some topic');
+      cy.get('#createOrUpdateCollectionButton').should('be.visible').should('not.be.disabled').click();
+      cy.contains(categoryName);
+      cy.contains('some topic');
+    });
+  });
+
+  describe('Should be able to add a tool to a category', () => {
+    it('be able to add tool to category', () => {
+      cy.visit(toolPath);
+      cy.get('#addToolToCollectionButton').should('be.visible').click();
+      cy.get('#addEntryToCollectionButton').should('be.disabled');
+      cy.get('#selectOrganization').click();
+      cy.get('mat-option').contains('Dockstore').click();
+      cy.get('#addEntryToCollectionButton').should('be.disabled');
+      cy.get('#selectCollection').click();
+      cy.get('mat-option').contains(categoryName).click();
+      cy.get('#addEntryToCollectionButton').should('not.be.disabled').click();
+    });
+  });
+
+  /*
+   * The following ITs were written before I discovered that the current
+   * UI2 CI process does not run Elasticsearch.  These tests will pass
```

Comment:
You can mock it https://docs.cypress.io/api/commands/intercept#Usage if you deem it worth the hassle, problably not though